### PR TITLE
Added Behavior related on Immutable tab and Conventional

### PR DESCRIPTION
**Pull request by adeliaferreira**
Hide tab immutable when no OSTree system's found - https://issues.redhat.com/browse/THEEDGE-3527

### CODE REVIEW

File: src/routes/InventoryTable.js [hunk 1]
```diff
@@ -231,6 +235,40 @@ const Inventory = ({
   };
 
   const EdgeParityEnabled = useFeatureFlag('edgeParity.inventory-list');
+  const INVENTORY_TOTAL_FETCH_URL_SERVER = '/api/inventory/v1/hosts';
+  const INVENTORY_TOTAL_FETCH_EDGE_PARAMS =
+    '?filter[system_profile][host_type]=edge';
```
[the remark this answers is not in the document]
Comment (adeliaferreira) [reply src/routes/InventoryTable.js]: Added filter, thanks

File: src/routes/InventoryTable.js [hunk 2]
```diff
@@ -231,6 +235,40 @@ const Inventory = ({
...
+  const INVENTORY_TOTAL_FETCH_CONVENTIONAL_PARAMS = '?page=1&per_page=1';
+
+  useEffect(() => {
+    if (EdgeParityEnabled) {
```
[the remark this answers is not in the document]
Comment (adeliaferreira) [reply src/routes/InventoryTable.js]: Extracted, thanks

File: src/routes/InventoryTable.js [hunk 3]
```diff
@@ -231,6 +235,40 @@ const Inventory = ({
   };
 
   const EdgeParityEnabled = useFeatureFlag('edgeParity.inventory-list');
+  const INVENTORY_TOTAL_FETCH_URL_SERVER = '/api/inventory/v1/hosts';
```
[the remark this answers is not in the document]
Comment (adeliaferreira) [reply src/routes/InventoryTable.js]: Constants moved, thanks

**Review by mkholjuraev — APPROVED** [review]
LGTM! A small possible improvement would be to think of how we can avoid making 2.

**Pull request merged**